### ISSUE TITLE
The 1080p Screensaver works but the 4K on remains completely black

### ISSUE THREAD

**Issue by Lucasfranz123321** [esetnik/Aerial#8]
I think I missed something I downloaded your Screensaver and installed it but I only get a black screen and no Pictures at all.
I would appreciate your help and love to use the 4K version which should look even greater.

**Comment by esetnik** [comment]
Are you running macOS 10.14 beta 1 by any chance? Many users are reporting issues with the brand new Mojave beta. Many users have been using my 4k Aerial for several months without issues and this is the first i've heard of black screensavers. Can you provide a full hardware profile?

**Comment by Lucasfranz123321** [comment]
No, I have my MacBook on 10.14 but this Maschine still runs Sierra 10.12.5 i think (latest Sierra) i think it May be That Sierra isn't able to Play HEVC/265 i think
If i Convert the dowloaded Videos in Cache to 264 will the screensaver recognize it?

**Comment by esetnik** [comment]
Yes that could certainly be the issue. You can verify by trying to play one of the cached videos in Quicktime. If it won't play in quicktime then it also won't play as a screensaver. You should be able to convert the videos to H264 and as long as you keep the naming identical they will work.

**Comment by Lucasfranz123321** [comment]
Yes those doesnt Play on my Maschine
Im ging top try it.
Regarding my MacBook pro the 4k Version lags quite a Bit. It runs 10.14 and its the Base model (mid 2014)  
Is this a Performance issue or would it Work on 10.13 ?

**Comment by esetnik** [comment]
Yeah it's likely that your Macbook pro lacks acceptable rendering performance to play HEVC encoded 4k video, this is a common complaint see #2, #3, #4.

**Comment by Lucasfranz123321** [comment]
Well @ least i can say it works great on the Beta. Only issue i found (but I guess thats an OS issue)  when i enter the preferences of your screensaver and I want to leave Afterwards, the "done" Button doesnt react anymore. I have to force quit to leave then. But still i guess thats an issue Apple hast to fix

**Comment by Lucasfranz123321** [comment]
I tried it now and it doesn't really work by entering those in the folder. Anyway I converted the files and let them play through an screensaver named "SaveHollywood" which basically just plays videos as the Screensaver and this works perfectly.

Quick Question will the new Earth screen saver for the Apple TV  also appear as a download through your Screensaver? 

Anyway thanks for your time and keep up the good work :)
